### PR TITLE
Dealing with environment variables

### PR DESCRIPTION
**Pull request by jonathangaldino**
This should help you deal with variables for each environment that you have.

Take a loot at backend/src/config/environment.js, each environment variable should be add to:

- .env file (wich you should copy from .env.example)
- environment array
- the config object that is exported

Doing that if any variable if forgotten will throw an error and the server won't start.

### CODE REVIEW

File: backend/src/server.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
-const express = require('express');
-const mongoose = require('mongoose');
+const express = require("express");
+const mongoose = require("mongoose");
 
-mongoose.connect('mongodb+srv://mikael:fimika13@mntbk-qboul.mongodb.net/test?retryWrites=true&w=majority', {
```
Comment (mikaelmessias) [inline backend/src/server.js:5]: Where will my connection string be added then? I get the basic idea, but I'm not sure about that

**Review by mikaelmessias — COMMENTED** [review]
I really appreciate your PR, but I have a little question before accepting it ... Please, take a look at the backend /src/server.js file

File: backend/src/server.js (hunk 1 above)
Comment (jonathangaldino) [reply backend/src/server.js:5]: In the commit #d780f34 I've added the new file called .env.example.

You will copy this file to a new one called .env (which won't be processed by git).
In the .env file, you should place the connection string and the port where the app will run.

**Comment by jonathangaldino** [comment]
I added a new commit, committing new files where you should put the variables in.

**Pull request merged**